pkg/ovs/openflow/interfaces.go

```diff
@@ -60,6 +61,9 @@ const (
 type Bridge interface {
 	CreateTable(id, next TableIDType, missAction MissActionType) Table
 	DeleteTable(id TableIDType) bool
+	CreateGroup(id GroupIDType) Group
```

Comment:
should this be `CreateGroupTable` / `DeleteGroupTable` as per OpenFlow specification terminology? I believe you are creating a group table here, to which later on you can add group entries.

Reply:
There is no "GroupTable" in OpenFlow protocol. The groups are "under" Switch directly.

Reply:
I see, seems that OVS only supports one group table, but multiple flow tables. Thanks for the clarification.